src/core/server/ui_settings/ui_settings_client.ts

```diff
@@ -77,40 +64,29 @@ export interface IUiSettingsClient {
  *  Service that provides access to the UiSettings stored in elasticsearch.
  *  @class UiSettingsService
  */
-export class UiSettingsService implements IUiSettingsClient {
-  private readonly _type: UiSettingsServiceOptions['type'];
-  private readonly _id: UiSettingsServiceOptions['id'];
-  private readonly _buildNum: UiSettingsServiceOptions['buildNum'];
-  private readonly _savedObjectsClient: UiSettingsServiceOptions['savedObjectsClient'];
-  private readonly _overrides: NonNullable<UiSettingsServiceOptions['overrides']>;
-  private readonly _getDefaults: NonNullable<UiSettingsServiceOptions['getDefaults']>;
-  private readonly _logWithMetadata: NonNullable<UiSettingsServiceOptions['logWithMetadata']>;
+export class UiSettingsClient implements IUiSettingsClient {
+  private readonly type: UiSettingsServiceOptions['type'];
+  private readonly id: UiSettingsServiceOptions['id'];
+  private readonly buildNum: UiSettingsServiceOptions['buildNum'];
+  private readonly savedObjectsClient: UiSettingsServiceOptions['savedObjectsClient'];
+  private readonly overrides: NonNullable<UiSettingsServiceOptions['overrides']>;
+  private readonly defaults: NonNullable<UiSettingsServiceOptions['defaults']>;
+  private readonly log: Logger;
 
   constructor(options: UiSettingsServiceOptions) {
-    const {
-      type,
-      id,
-      buildNum,
-      savedObjectsClient,
-      // we use a function for getDefaults() so that defaults can be different in
-      // different scenarios, and so they can change over time
```

Comment:
It looks like a misleading comment. The platform calculates values once https://github.com/restrry/kibana/blob/b7114c7df6a547fbab769079589b5f755dc709bf/src/legacy/ui/ui_settings/ui_settings_mixin.js#L31
Dynamic behavior removed in https://github.com/elastic/kibana/commit/c65da14d7c509bd3b85056a87c651674c0f353dd#diff-a23948121f4bc50ced65723c687f343d